labonneboite/web/app.py

```diff
@@ -251,6 +248,9 @@ def create_app():
     register_teardown_appcontext(flask_app)
     register_teardown_appcontext(flask_app)
 
+    # Emails.
+    mailjet.init_mail(mailjet_api_key=settings.MAILJET_API_KEY, mailjet_api_secret=settings.MAILJET_API_SECRET, form_email=settings.FORM_EMAIL, email_default_from=settings.EMAIL_DEFAULT_FROM)
```

Comment:
Line too long... split it please.

Pour info on a du pylint que malheureusement on a laissé dépérir cf `make pylint-all`. Et aussi un precommit hook pour juste pylinter les fichiers effectivement modifiés dans ton commit:

```
cat .git/hooks/pre-commit
#!/bin/bash
cd ~/lbb/labonneboite && \
git-pylint-commit-hook && \
cd ~/lbb/labonneboite && \
#make test_api && \
#make test_app && \
#make test_importer && \
#make test_integration && \
echo all right
```

Reply:
Sur itou on utilise un simple `make black` qui fait les changements pour nous et je suis fan.